src/core/utils/positions.js

```diff
@@ -0,0 +1,30 @@
+export function parsePosition(value) {
+  switch (typeof value) {
+    case 'number':
+      return {
+        position: value,
+        relative: false
+      };
+
+    case 'string':
+      const PERCENT_OR_PIXELS_REGEX = /([01]?[0-9]?[0-9])(%|px)/;
```

Comment:
- No `33.3%`?
- Define outside?

Reply:
Might be a preference - I think it'll be easier to read if we keep the regex short and forgiving, and validate the parsed number instead.

Reply:
Good points. Addressed.